crates/astria-composer/tests/blackbox/composer.rs

```diff
@@ -54,6 +60,46 @@ async fn tx_from_one_rollup_is_received_by_sequencer() {
     .expect("mocked sequencer should have received a broadcast message from composer");
 }
 
+#[tokio::test]
+async fn tx_from_one_rollup_is_received_by_sequencer_from_grpc_collector() {
+    let test_composer = spawn_composer(&[]).await;
+    tokio::time::timeout(
+        Duration::from_millis(100),
+        test_composer.setup_guard.wait_until_satisfied(),
```

Comment:
not for this PR: since this setup guard is likely the same for all tests, you can just move this into `spawn_composer` so that readers of the test don't have to worry about this (since it's kinda orthogonal to the aim of this PR).